src/share/cxx/prim_driver.cpp

```diff
@@ -70,12 +70,7 @@ HybridVCoord  Errors::runtime_abort("CAM forcing not yet availble in C++.\n"
                         Errors::err_not_implemented);
   // call TimeLevel_Qdp(tl, qsplit, n0_qdp)
 
-  // if (ftype==0) then
-  //   call t_startf("ApplyCAMForcing")
-  //   call ApplyCAMForcing(elem, fvm, hvcoord,tl%n0,n0_qdp, dt_remap,nets,nete)
-  //   call t_stopf("ApplyCAMForcing")
-
-  // elseif (ftype==2) then
+  // if (ftype==2) then
```

Comment:
harmless but i wonder why you wanted to touch this, there will be ftype0 and ftype2 when CAM is on.

Reply:
oops, it's all commented; I just meant to get rid of the commented code entirely